### PR TITLE
Add missing NEXT_MAJOR tags

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: Block/LocaleSwitcherBlockService.php [hunk 1]
```diff
@@ -23,6 +23,8 @@
 class LocaleSwitcherBlockService extends BaseBlockService
 {
     /**
+     * NEXT_MAJOR: remove this method.
+     *
      * @deprecated Will be removed when upgrading to SonataBlockBundle 3
```
Comment (OskarStark) [inline Block/LocaleSwitcherBlockService.php]: the deprecation message looks wrong, can you adjust this @greg0ire ?

**Comment by OskarStark** [comment]
Can you please fix StyleCI @greg0ire ?

**Comment by greg0ire** [comment]
Fixed @OskarStark

**Comment by OskarStark** [comment]
Thank you @greg0ire 👍

**Pull request merged**